data/brands/amenity/restaurant.json

```diff
@@ -2613,6 +2613,19 @@
         "name": "Ristorante Del Arte"
       }
     },
+    {
+      "displayName": "Rolls",
+      "id": "rolls-fe858e",
+      "locationSet": {"include": ["fi"]},
+      "tags": {
+        "amenity": "restaurant",
+        "brand": "Rolls",
+        "brand:wikidata": "Q3195068",
+        "brand:wikipedia": "en:Rolls (restaurant chain)",
```

Comment:
If it's a finish brand we should use finish Wikipedia

Reply:
I have thought the same as well but when I searched for Rolls in Finnish Wikipedia the page doesn't exist, only within the English Wikipedia that you can find that restaurant.

Reply:
The [wikidata page for Rolls](https://www.wikidata.org/wiki/Q3195068) shows that there is one:
(this is where I usually go to look it up)

<img width="385" alt="Screen Shot 2020-10-06 at 9 57 01 AM" src="https://user-images.githubusercontent.com/38784/95211234-4fdb0f80-07ba-11eb-87e6-bca7a9571136.png">

Reply:
🤦🏻‍♂️ Thank you, I’ll get that updated.